rossum/password.py

```diff
@@ -23,3 +23,20 @@ def change_command(ctx: click.Context, password: str) -> None:
                 return click.echo(
                     'Please exit and run "rossum configure" to update existing credentials.'
                 )
+
+
+@cli.command(name="reset", help="Reset password. Please specify an email address.")
+@click.argument("email")
+@click.pass_context
+def reset_command(ctx: click.Context, email: str) -> None:
+    """
+    Reset password for a user by specifying an user email.
+    """
+    if email:
+        with RossumClient(context=ctx.obj) as rossum:
+            result = rossum.reset_user_password(email)
+            return click.echo(result.get("detail"))
```

Comment:
Other `rossum` commands do not return anything. They only echo. Consider following the syntax of the already existing commands.